servicemesh-install/topics/install.adoc

```diff
@@ -230,16 +230,11 @@ The OpenShift Master Public URL must be configured to match the public URL of yo
 
 The previous commands create a new deployment within the `istio-operator` project and run the operator responsible for managing the state of the {ProductName} control plane through the custom resource.
 
-. To verify that the operator is installed correctly, execute the following command:
-+
-```
-$ oc get pods -n istio-operator
-```
-+
-. You can access the logs from the `istio-operator` pod with the following command, replacing `<pod name>` with the name of the pod you discovered.
+. To verify that the operator is installed correctly, access the logs from the operator pod using the following command
```

Comment:
s/using the following command/by running the following command:

Reply:
I'll make this change and repush

Reply:
Done and pushed